src/main/java/edu/usf/cutr/gtfsrtvalidator/api/model/SessionModel.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2017 University of South Florida.
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.usf.cutr.gtfsrtvalidator.api.model;
+
+import javax.persistence.*;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.io.Serializable;
+
+@XmlRootElement
+@Entity
+@Table(name = "Session")
+public class SessionModel implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    @Column(name = "sessionId")
+    private int sessionId;
+
+    @ManyToOne
+    @JoinColumn(name = "rtFeedId")
+    private GtfsRtFeedModel gtfsRtFeedModel;
+
+    @Column(name = "startTime")
+    private long sessionStartTime;
+
+    @Column(name = "endTime")
+    private long sessionEndTime;
+
+    @Column(name = "userId")
```

Comment:
Let's call this the same thing across both the server and Javascript code - right now it's `userId` server-side, and `clientId` in Javascript.  I think `clientId` is a more accurate name, so let's use that server-side as well.